### DISCUSSION TITLE
Missing docker slim-2.5.0-python3.10

### DISCUSSION THREAD

**Discussion by raphaelauv** [apache/airflow#28082] [General] [answered]
the tag slim-2.5.0-python3.10 is missing on dockerhub

https://hub.docker.com/r/apache/airflow/tags?page=1&name=slim-2.5.0-python3

**Comment by potiuk** [comment, marked as answer]
Yes.  thanks @raphaelauv  for noticing, 

We have some intermittent/flaky failures of ARM builds on Amazon ARM instances which need taking a closer look. 

I restarted it now https://github.com/apache/airflow/actions/runs/3603432288/jobs/6080260821 - I hope it will succeed this time (if not I will manually push it) later today.

**Reply by potiuk** [reply]
Ok. It succeeded this time so it should be available @raphaelauv
